cpp/test/util_test/from_file_test.cc

```diff
@@ -90,17 +99,8 @@ TYPED_TEST(FromFileTest, CSVTest) {
   }
 }
 
-TYPED_TEST(FromFileTest, IfFileNotExistsRowsAndColsMethod) {
-  ASSERT_DEATH(
-      {
-        this->Filer(
-            "../test/data_for_test/matrix_not_exist.txt", 2, 2, ' ');
-      }
-      , "Cannot open file .*, exiting...");
-}
-
-TYPED_TEST(FromFileTest, IfFileExistsRowsAndColsMethod) {
-  this->Filer("../test/data_for_test/matrix_2_2.txt", 2, 2, ' ');
+TYPED_TEST(FromFileTest, CSVTest) {
+  this->Filer("../test/data_for_test/matrix_2_2_csv.txt", ",");
```

Comment:
could you rename all *_csv.txt files to *.csv files (e.g. matrix_2_2_csv.txt -> matrix_2_2.csv)? The extension should be .csv for csv files.